nfs/inc/rpc_task.h

```diff
@@ -1772,6 +1808,14 @@ struct rpc_task
                      fuse_ino_t parent_ino);
     void run_lookup();
 
+    void init_proxy_lookup(fuse_req *request,
+                     const char *name,
+                     fuse_ino_t parent_ino,
+                     enum fuse_opcode proxy_optype,
+                     fuse_file_info *fileinfo = nullptr);
+
```

Comment:
since proxy task is created on behalf of an original task, the only argument this should take is the original task.